kubernetes/cray-nexus/values.yaml

```
@@ -189,6 +189,14 @@ setup:
     enabled: true
     secret: nexus-admin-credential
 
+hooks:
+  preInstall:
+    image:
+      repository: artifactory.algol60.net/csm-docker/stable/docker-kubectl
+      tag: 1.19.15
```

This tag should be 1.24.17 because that is the version of kubernetes in CSM 1.6.
```suggestion
      tag: 1.24.17
```